invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/dropdown.overrides

```diff
@@ -78,7 +78,45 @@
   }
 }
 
-.mobile-table-cell .ui.dropdown{
+.members-dropdown-container {
+  .ui.dropdown {
+    border: @selectionBorder;
+    padding: @selectionPadding;
+    border-radius: @selectionBorderRadius;
+
+    .menu {
+      overflow-x: auto;
+      overflow-y: hidden;
+    }
+
+    .active.selected.item {
+      font-weight: normal;
+
+    }
+
+    .item {
+      .disabled.icon {
+        visibility: hidden;
```

Comment:
could we instead of hiding it with custom CSS,  not render the icon at all ?

Reply:
completely right :smile: